engine/runtime-integration-tests/src/test/java/org/enso/compiler/ExecCompilerTest.java

```diff
@@ -408,16 +408,15 @@ public void testDoubledRandom() throws Exception {
           polyglot java import java.util.Random
 
           run seed =
-              operator1 = Random.new_generator seed
+              Random.new_generator seed
           """);
     var run = module.invokeMember("eval_expression", "run");
     try {
       var err = run.execute(1L);
-      fail("Not expecting any result: " + err);
+      assertTrue("Returned value represents exception: " + err, err.isException());
+      throw err.throwException();
```

Comment:
Resolving `polyglot java import`s lazily prevents us to emit error during compilation. Instead we get a (dataflow) `Error` when we execute the `run` function and try to resolve `Random` function. Such an error is then returned as a value and needs to be `throwException` manually.